src/types/Params.ts

```diff
@@ -0,0 +1,3 @@
+export interface UserRequestOptions {
```

Comment:
Can we call this `RequestOptions`? Calling it `UserRequestOptions` might imply some relation to users specifically

Reply:
`WarrantRequestOptions` might work too

Reply:
Yeah, I see what you're saying. Maybe `ApiRequestOptions`?

Reply:
Renamed to `WarrantRequestOptions`